source/git/gitsource.go

```diff
@@ -484,6 +594,13 @@ func git(ctx context.Context, dir string, args ...string) (*bytes.Buffer, error)
 			"GIT_TERMINAL_PROMPT=0",
 			//	"GIT_TRACE=1",
 		}
+		if sshAuthSock != "" {
+			cmd.Env = append(cmd.Env, "SSH_AUTH_SOCK="+sshAuthSock)
+		}
+		if knownHosts != "" {
+			cmd.Env = append(cmd.Env, "GIT_SSH_COMMAND=ssh -o UserKnownHostsFile="+knownHosts)
```

Comment:
what options could we set to make sure we never use the global ssh config and only the credentials that are passed with the build?

Reply:
good idea, we can set `-F /dev/null` to avoid the global ssh config.